### PR TITLE
[Urgent] add posters with executive publishing group  to browse pools page

### PR DESCRIPTION
**Pull request by brindasasi**
🤖 Resolves {#8607}

## 👋 Introduction

Describe what this PR is solving.

## 🕵️ Details

Add any additional details that could assist with reviewing or testing this PR.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Create a Pool with Executive Stream and Executive Publishing group 
2. Publish it 
3. Go to browse pools page and see the pool card exists for the same 
4. go to en/executive page and see the pool card exists there as well 
5. Click on the pool card to open up the job poster and verify you get the proper education requirement section as well

### CODE REVIEW

File: apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.test.tsx [hunk 1]
```diff
@@ -98,23 +104,27 @@ describe("BrowsePoolsPage", () => {
     );
   });
 
-  it("should only show IT jobs", async () => {
+  it("should only show IT and Executive jobs", async () => {
     renderBrowsePoolsPage({
-      pools: [publishedItJobsPool, publishedExecJobsPool],
+      pools: [publishedItJobsPool, publishedExecJobsPool, publishedIAPJobsPool],
```
Comment (mnigh) [inline apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.test.tsx:109]: If the pool `publishedIAPJobsPool` is being added, why is the test called **should only show IT and Executive jobs**?

File: apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.tsx [hunk 2]
```diff
@@ -64,7 +64,8 @@ export const BrowsePools = ({ pools }: BrowsePoolsProps) => {
   const activeRecruitmentPools = pools.filter(
     (p) =>
       p.status === PoolStatus.Published && // list jobs which have the PUBLISHED PoolStatus
-      p.publishingGroup === PublishingGroup.ItJobs, // and which are meant to be published on the IT Jobs page
+      (p.publishingGroup === PublishingGroup.ItJobs ||
+        p.publishingGroup === PublishingGroup.ExecutiveJobs), // and which are meant to be published on the IT Jobs page
```
Comment (mnigh) [inline apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.tsx:68]: I don't think this comment makes sense: there is no IT Jobs page.
```suggestion
        p.publishingGroup === PublishingGroup.ExecutiveJobs),
```

**Review by mnigh — COMMENTED** [review]
It does render posters with `ExecutiveJobs` publishing group on the browse pools page, but a few questions about the change to the test and a comment that can probably be removed.

## Screenshots
Screen Shot 2023-11-28 at 14 07 05

Screen Shot 2023-11-28 at 14 07 57

File: apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.test.tsx (hunk 1 above)
Comment (brindasasi) [reply apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.test.tsx:109]: Just to verify IAP job is not displayed in here

Comment (vd1992) [reply apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.test.tsx:109]: Yeah, the test verifies that it gets filtered out

**Review by vd1992 — APPROVED** [review]
Works for me, good idea with testing the other publishing group case. 
Just need to update the comment Matt highlighted to be more accurate

Exec streams, publishing group, and added classification. 

image

**Review by mnigh — APPROVED** [review]

**Pull request merged**